x-pack/test/apm_api_integration/tests/mobile/mobile_main_statistics_by_field.spec.ts

```diff
@@ -82,16 +201,14 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         });
         const fieldValues = response.mainStatistics.map((item) => item.name);
 
-        expect(fieldValues).to.be.eql(['1.1', '1.2', '2.3']);
+        expect(fieldValues).to.be.eql(['1.0', '2.0']);
```

Comment:
Either do as above:

```suggestion
        expect(fieldValues).to.be.eql(SERVICE_VERSIONS);
```

Or alternatively

```suggestion
        expect(fieldValues).to.be.eql([galaxyServiceVersion, huaweiServiceVersion]);
```